webapp/src/Entity/Contest.php

```diff
@@ -988,9 +976,13 @@ public function getAbsoluteTime(?string $time_string)
             // Take into account the removed intervals.
             /** @var RemovedInterval[] $removedIntervals */
             $removedIntervals = $this->getRemovedIntervals()->toArray();
-            usort($removedIntervals, function (RemovedInterval $a, RemovedInterval $b) {
-                return Utils::difftime((float)$a->getStarttime(), (float)$b->getStarttime());
-            });
+            usort(
```

Comment:
I think this is harder to read.

Reply:
Because it’s multi line? I’m not sure I like the long lines either

Reply:
What about:
```
            usort(
                $removedIntervals,
                static fn(RemovedInterval $a, RemovedInterval $b) => Utils::difftime(
                    (float)$a->getStarttime(),
                    (float)$b->getStarttime()
                )
            );
```

Also not super readable though.

Alternative is
```
            usort(
                $removedIntervals, static fn(RemovedInterval $a, RemovedInterval $b) =>
                    Utils::difftime((float)$a->getStarttime(), (float)$b->getStarttime())
            );
```

but also a bit weird.

Or we leave it as the non-short option.

Reply:
I think we should just leave it as it was

Reply:
I would be fine with the old situation (although its the only place where we use the non-short now?) or the last alternative (although it indeed looks a bit weird)

Reply:
I reverted this one and two other ones to the old format, since that's indeed beter.